README.md

```diff
@@ -39,16 +29,41 @@ Install with [Carthage](https://github.com/Carthage/Carthage) by adding the foll
 github "square/in-app-payments-ios"
 ```
 
-## Add build phase to setup the SquareInAppPaymentsSDK and/or SquareBuyerVerificationSDK framework ##
+### 2. Add build phase to setup the SquareInAppPaymentsSDK and/or SquareBuyerVerificationSDK framework ###
 After adding the framework using any of the above methods, follow the below instructions to complete the setup. 
 
 On your application targets’ Build Phases settings tab, click the + icon and choose New Run Script Phase. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:
+
 ```
 FRAMEWORKS="${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}"
 "${FRAMEWORKS}/SquareInAppPaymentsSDK.framework/setup"
 ```
 
+## Swift Package Manager
+
+[Swift Package Manager](https://www.swift.org/package-manager/) support is available for Swift 5.3 and above but is currently not the recommended method for using Square In-App Payments.
+
+### 1. Add the following to your top-level `dependencies` block within your `Package.swift`:
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/square/in-app-payments-ios", .upToNextMajor(from: "1.6.0")),
+]
+```
+
+### 2. Add build phase to setup the SquareInAppPaymentsSDK and/or SquareBuyerVerificationSDK framework ###
+
+On your application targets’ Build Phases settings tab, click the + icon and choose New Run Script Phase. Create a Run Script in which you specify your shell (ex: /bin/sh), add the following contents to the script area below the shell:
+
+```
+SETUP_SCRIPT="${CONFIGURATION_BUILD_DIR}/SquareInAppPaymentsSDK.framework/setup"
+if [ -f "$SETUP_SCRIPT" ]; then
+  "$SETUP_SCRIPT"
+fi
+```
+
+**Note:** Because of the build phase script, when switching between a debug build and release build (archiving), deleting the Derived Data may be necessary to prevent errors during building or releasing.
```

Comment:
A few alternative ways we could put this, just to explore some options with wording:

> Important Reminder: Due to the presence of the build phase script, you may need to delete the Derived Data folder when transitioning between a debug build and a release build (archiving). This will help prevent any potential errors during the build or release process.

> Attention: To ensure a smooth transition between debug and release builds (archiving), it's recommended to remove the Derived Data folder. This is due to the build phase script and helps avoid any build or release errors.

> Heads Up: When toggling between debug and release builds (archiving), it's important to clear the Derived Data folder to prevent potential issues during the build or release process. This is necessary because of the build phase script.

> Note: The build phase script requires you to delete Derived Data folder when switching from a debug build to a release build (archiving) or vice versa. This helps prevent errors during building or releasing.

> Pro Tip: Ensure a hassle-free build or release process by removing the Derived Data folder when changing between debug and release builds (archiving). This is essential due to the build phase script used.

> Take Note: The build phase script in the Swift SDK necessitates the deletion of the Derived Data folder when transitioning between debug and release builds (archiving). Doing so will help avoid any complications during the build or release process.